frontends/web/src/components/transactions/transaction.jsx

```diff
@@ -111,26 +113,22 @@ export default class Transaction extends Component {
         );
         const sign = ((type === 'send') && '−') || ((type === 'receive') && '+') || null;
         const sDate = time ? this.parseTimeShort(time) : '---';
-        const statusStyle = {
-            complete: style.statusIndicatorComplete,
-            pending: style.statusIndicatorPending,
-            failed: style.statusIndicatorFailed,
-        }[status];
         const statusText = {
             complete: 'Complete',
-            pending: 'Pending',
+            pending: `Pending`,
```

Comment:
While you're at it making changes here, let's move all three of these to the i18n file?